### PR TITLE
💿 Latest goflow 

### PR DESCRIPTION
**Pull request by rowanseymour**
To address #200 and #192

### CODE REVIEW

File: models/env.go [hunk 1]
```diff
@@ -18,9 +19,12 @@ import (
 
 // Register a airtime service factory with the engine
 func init() {
+	// give airtime transfers an extra long timeout
+	httpClient := &http.Client{Timeout: time.Duration(120 * time.Second)}
```
Comment (rowanseymour) [inline models/env.go:23]: how does 2 minutes sound for DT One?

File: goflow/engine.go [hunk 2]
```diff
@@ -64,9 +68,13 @@ func Engine() flows.Engine {
 // Simulator returns the global engine instance for use with simulated sessions
 func Simulator() flows.Engine {
 	simulatorInit.Do(func() {
+		webhookHeaders := map[string]string{
+			"User-Agent":      "RapidProMailroom/" + config.Mailroom.Version,
+			"X-Mailroom-Mode": "simulation",
```
Comment (rowanseymour) [inline goflow/engine.go:73]: So many different ways to convey that the request is part of simulation.. could be part of the User-Agent.. should it be called Mailroom or Temba or RapidPro.. opted for this over `X-Mailroom-Simulation: 1` as it's more extendable.

File: models/assets.go [hunk 3]
```diff
@@ -439,3 +439,7 @@ func (a *OrgAssets) ResthookBySlug(slug string) *Resthook {
 func (a *OrgAssets) Templates() ([]assets.Template, error) {
 	return a.templates, nil
 }
+
+func (a *OrgAssets) Globals() ([]assets.Global, error) {
+	return nil, nil // not supported yet
```
Comment (rowanseymour) [inline models/assets.go:444]: need to add the models on the django side, then wire this up, then add the UI on the django side

**Comment by rowanseymour** [comment]
@nicpottier note the coverage change for `hooks/email_created`.. which I assume is unrelated but seems that test isn't doing what it used to do

**Comment by rowanseymour** [comment]
@nicpottier actually that's back to normal.. an intermittent thing evidently

File: models/env.go (hunk 1 above)
Comment (nicpottier) [reply models/env.go:23]: Better than 10 minutes? :)

**Review by nicpottier — APPROVED** [review]

File: models/env.go (hunk 1 above)
Comment (rowanseymour) [reply models/env.go:23]: I just can't believe an HTTP request ever came back after 10 minutes.. I can't

**Pull request merged**